x-pack/plugins/infra/public/alerting/log_threshold/rule_data_formatters.ts

```diff
@@ -6,16 +6,32 @@
  */
 
 import { ALERT_REASON, ALERT_START } from '@kbn/rule-data-utils';
-import { ObservabilityRuleTypeFormatter } from '@kbn/observability-plugin/public';
-import { getLogsAppAlertUrl } from '../../../common/formatters/alert_link';
+import type { ObservabilityRuleTypeFormatter } from '@kbn/observability-plugin/public';
+import type { LocatorPublic } from '@kbn/share-plugin/public';
+import type { LogsLocatorParams } from '../../../common/locators';
 
-export const formatRuleData: ObservabilityRuleTypeFormatter = ({ fields }) => {
-  const reason = fields[ALERT_REASON] ?? '';
-  const alertStartDate = fields[ALERT_START];
-  const timestamp = alertStartDate != null ? new Date(alertStartDate).valueOf() : null;
+export const formatRuleData: (
```

Comment:
**nit:** as this function doesn't directly format the rule data, we could rename it something like `createRuleFormatter` to know it returns a different function.